parser/CMakeLists.txt

```diff
@@ -129,3 +133,6 @@ new_ec_test(empty_editorconfig_file empty.in test4 "^[ \t\n\r]*$")
 new_ec_test(newlines_only_editorconfig_file newlines_only.in test4 "^[ \t\n\r]*$")
 new_ec_test(comments_only_editorconfig_file comments_only.in test4 "^[ \t\n\r]*$")
 new_ec_test(comments_and_newlines_editorconfig_file comments_and_newlines.in test4 "^[ \t\n\r]*$")
+
+# A key set to empty string
```

Comment:
Do you mean "value" set to empty string? This is the only confusing spot, otherwise the PR looks good.